<launch>
  <node name="" pkg="rosserial_python" type="serial_node.py" args="/dev/ttyACM0"/>
  <node name="call_sensors_server" pkg="maccepavd" type="sensor_server.py"/>
</launch>
